### ISSUE TITLE
How to load images ?

### ISSUE THREAD

**Issue by Ryoumiya** [iperov/litenn#3]
Hello,
I been playing around with LiteNN, and i wanted to say great job on the LiteNN. also i want to ask how to load images as there seem no _official_ way to load batches of images, my current stop gap measure is to use opencv to load images and convert it into numpy array afterward convert it into a Tensor.

```
imgcv = cv2.imread(imgpath)
test_tensor = nn.Tensor_from_value(imgcv)
```

Will there be a way to load images directly from LiteNN  ?,

**Comment by iperov** [comment]
LiteNN receives only numpy data

**Comment by Ryoumiya** [comment]
yes sorry, what wanted to ask is _will there be in inbuilt function to load images similar to pytorch "DataLoader" ?_

as for the OpenCV, it is just the easiest way i know to load images as numpy array.

**Comment by iperov** [comment]
why? just use cv2 imread , or lod through PIL
